src/main/java/ch/bfh/bti7081/view/NewSeminarView.java

```diff
@@ -153,9 +178,11 @@ private void addBindingToForm() {
 
     private void setFormActions() {
         save.addClickListener(event -> {
+            UserDTO sessionUser = (UserDTO) VaadinSession.getCurrent().getAttribute("user");
+
             if (binder.writeBeanIfValid(newSeminar)) {
                 try {
-                    presenter.sendSeminarToBackend(newSeminar);
+                    presenter.sendSeminarToBackend(newSeminar, sessionUser);
```

Comment:
Warum wird hier der sessionuser mitgegeben? Kann der nicht im Backend abgefragt werden? Wäre sicherer, weil sonst ein potentieller Hacker den User "fälschen" kann und sich  so Zugang verschaffen.

Reply:
Der SessionUser wird serverseitig in der VaadinSession gespeichert. Ist es möglich, das zu fälschen? Das wären die Alternativen, aber meiner Ansicht haben die gleichen Probleme oder sind noch einfacher fälschbar:
- nur Username ins Backend mitgeben (meiner Ansicht nach noch einfacher fälschbar)
- so wie sonst überall auch den User anhand des Namens neu laden (gleiche Problematik wie sessionUser)